sdv/single_table/base.py

```diff
@@ -99,6 +107,14 @@ def __init__(
     ):
         self._validate_inputs(enforce_min_max_values, enforce_rounding)
         self.metadata = metadata
+        if isinstance(metadata, Metadata):
+            self.metadata = metadata._convert_to_single_table()
+            self.real_metadata = metadata
```

Comment:
I think I'll remove this attribute and just convert it to the new metadata when calling get_metadata